### PR TITLE
Add fill machine on Emi

### PR DESCRIPTION
**Pull request by Arborsm**
Temporary....

### CODE REVIEW

**Comment by Arborsm** [comment]
Before AE2 add native Emi compact? #649

File: common/src/main/java/com/gregtechceu/gtceu/core/mixins/emi/AddEmiRecipeHandler.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import java.util.List;
+
+@Mixin(EmiRecipeFiller.class)
+public class AddEmiRecipeHandler {
```
Comment (screret) [inline common/src/main/java/com/gregtechceu/gtceu/core/mixins/emi/AddEmiRecipeHandler.java]: Should be called `EmiRecipeHandlerMixin`

**Review by screret — CHANGES_REQUESTED** [review]
Also: why include an AE2 handler in this?

**Comment by Arborsm** [comment]
> Also: why include an AE2 handler in this?

for ae2 filling,I don't know why but Gtm's recipes have some issue that emi's bridge between jei and ae2 not work

**Comment by screret** [comment]
Lets try waiting for the native AE2 solution first will we?

**Comment by Arborsm** [comment]
> Lets try waiting for the native AE2 solution first will we?

hm,then I deleted those ae things

**Comment by Arborsm** [comment]
done

**Review by screret — APPROVED** [review]

**Review by screret — APPROVED** [review]

**Pull request merged**